Test/RivetTest/Functions/Clear-TestDatabase.ps1

```diff
@@ -22,7 +22,7 @@ function Clear-TestDatabase
 
     if( Test-Database -Name $Name )
     {
-        $query = "select * from [$($Name)].[rivet].[Migrations] where ID > 1000000000 order by ID"
+        $query = "select * from [$($Name)].[rivet].[Migrations] where ID > 10101000000 order by ID"

```

Comment:
Use a module-level variable for this magic number and update the other places that use it to use the variable.